plugins/dataset_feature_space.cc

```diff
@@ -114,6 +114,9 @@ getColumnInfo(std::shared_ptr<Dataset> dataset,
                 auto categorical = std::make_shared<ML::Fixed_Categorical_Info>(allValues);
                 result.info = ML::Feature_Info(categorical);
             }
+            else if (types.size() == 1 && types.begin()->first == CellValue::EMPTY) {
```

Comment:
When I tested, the value type I got was `EMPTY`. If I switch the comparison to `UNKNOWN`, I doubt this will still work.